nobl9/data_source.go

```diff
@@ -0,0 +1,66 @@
+package nobl9
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/nobl9/nobl9-go/manifest/v1alpha"
+)
+
+func dataSourceAWSIAMRoleAuthExternalID() *schema.Resource {
+	return &schema.Resource{
+		Description: "[Cross account IAM roles](https://docs.nobl9.com/Sources/Amazon_CloudWatch/#cross-account-iam-roles-new)",
```

Comment:
Just having link is kinda ugly... When reading our terraform docs it makes it annoying to open 353265423 links to get anything about particular resource or data source. At least some "This returns external ID and AWS account ID that can be used to create cross-account IAM roles in AWS." would be nice.

Reply:
Good point :)